modules/installation-configuration-parameters.adoc

```diff
@@ -591,6 +591,10 @@ Optional installation configuration parameters are described in the following ta
 |Extends the set of optional capabilities beyond what you specify in `baselineCapabilitySet`. You may specify multiple capabilities in this parameter.
 |String array
 
+|`cpuPartitioningMode`
+|Enables workload partitioning, which isolates {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs. Workload partitioning can only be enabled during installation and cannot be disabled after installation.
```

Comment:
Suggest to add below statement from openshift-install explain doc, since it still needs additional configuration to make workload partitioning work.
`When this field is set the cluster will be flagged for CPU Partitioning allowing users to segregate workloads to specific CPU Sets. This does not make any decisions on workloads it only configures the nodes to allow CPU Partitioning. `

@sunilcio wdyt?

Reply:
@jinyunma yes, I agree. It is good idea to include this statement also. Thanks!